<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nueva Receta</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color : #FFF;
	margin : 0px;
	padding : 0px;
}

QFrame#content {
	background-color : transparent;
	border: 1px solid #DADDE1;
	border-radius: 5px;
	min-width : 400px;
	min-height : 460px;
}

QLabel {
	font : 77 12px &quot;Arial&quot;;
	color : green;
}

QLineEdit, QComboBox, QTextEdit{
	background-color : transparent;
	border : 1px solid #DADDE1;
	border-radius : 5px;
	color : #616161;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="content">
     <property name="minimumSize">
      <size>
       <width>402</width>
       <height>462</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>* Complete los campos para agregar la nueva receta.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="tfd_name">
        <property name="placeholderText">
         <string> * Ingrese su nombre</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tfd_time1">
        <property name="placeholderText">
         <string> * Tiempo de preparación en minutos</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tfd_time2">
        <property name="placeholderText">
         <string> * Tiempo de cocción en minutos</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ingredients">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color : #6BA6FD;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #DBE9FF;
	padding-left : 10px;
	padding-right : 10px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #6BA6FD;
	border: 1px solid #6BA6FD;
	border-radius: 5px;
}
</string>
        </property>
        <property name="text">
         <string>Ingredientes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="text_area">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string> * Instrucciones</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Mostrar productos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : red;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : red;
	border: 1px solid red;
	border-radius: 5px;
}

</string>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_accept_and_new">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Mostrar productos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #42B72A;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #42B72A;
	border: 1px solid #42B72A;
	border-radius: 5px;
}

</string>
           </property>
           <property name="text">
            <string>Agregar más</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_accept">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Mostrar productos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #42B72A;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #42B72A;
	border: 1px solid #42B72A;
	border-radius: 5px;
}

</string>
           </property>
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
